products/freebsd.md

```diff
@@ -134,14 +146,11 @@ releases:
 
 ---
 
-> [FreeBSD](https://www.freebsd.org) is an operating system used to power modern servers, desktops, and embedded platforms.
-
-Each release is supported by the Security Officer for a limited time only. Under the current support model, each major version's stable branch is explicitly supported for 5 years, while each individual point release is only supported for three months after the next point release.
-
-The Expected EoL (end-of-life) column indicates the earliest date on which support for that branch or release will end. Please note that these dates may be pushed back if circumstances warrant it.
+> The [FreeBSD](https://www.freebsd.org/) operating system powers modern servers, desktops, and embedded platforms.
 
-The FreeBSD Security Officer provides security advisories for `-STABLE` Branches and the Security Branches. Advisories are not issued for the `-CURRENT` Branch, which is primarily oriented towards FreeBSD developers.
+[The support model](https://www.freebsd.org/security/#sup), summarised:
 
-The -STABLE branch tags have names like `stable/10`. The corresponding builds have names like `FreeBSD 10.1-STABLE`.
+* each `stable` branch has a five-year lifespan
+* whenever there is a new `RELEASE` from a `stable` branch, [Security Officer](https://www.freebsd.org/administration/#t-secteam) support for the prior `RELEASE` from the same branch will end three months later.
 
-Each FreeBSD Release has an associated Security Branch. The Security Branch tags have names like `releng/10.1`. The corresponding builds have names like `FreeBSD 10.1-RELEASE-p4`.
+[freebsd-version(1)](https://www.freebsd.org/cgi/man.cgi?query=freebsd-version&sektion=1&manpath=FreeBSD)
```

Comment:
I think we should keep the definition of `stable` and `releng`. I do not know FreeBSD and I had to look at https://www.freebsd.org/security/#sup to know what that is.

Reply:
@grahamperrin, @usta, I added a commit to :

- Revert the product description to what can be found on https://www.freebsd.org/. Usually the product description is a copy-paste of what can be found on the product landing page.
- Simplify blurb by combining what I found on https://www.freebsd.org/security/#model (with an explanation of what is `stable/x` and what is `releng/x.y`) and a small paragraph explaining that the EOL date are not exact. All the details mentioning the security officers have been removed: if EndOfLife users want details they can go to the _More information_ link : https://www.freebsd.org/security/.

What do you think of this new description ?

Reply:
> … to know what `releng` means.

A good question. 

----

[FreeBSD Release Engineering](https://docs.freebsd.org/en/articles/freebsd-releng/) – an _article_ in the documentation portal – begins by describing itself as outdated. 

Neither the _Documentation_ menu, nor the footer, includes a link to FAQ. This should be fixed. 

[Release Engineering Information](https://www.freebsd.org/releng/) – a _FreeBSD Project_ page – is better, but not ideal. There's mention of `stable` and `releng`, without explaining that `stable` is (for many use cases) **not** best for stability; and so on. 

Again, no direct link to FAQ; it's nested in the _Documentation_ menu. I'd like to change this, to raise awareness of existence of FAQ, but not just yet (see below, the rewrite). 

The [home page](https://www.freebsd.org/) does include a direct link to FAQ, however the design of the page causes links such as these to be overlooked. 

Existing FreeBSD bugs include [265276 – Books: FAQ: Single HTML: 404](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=265276).

The FAQ page mentions _release engineering_ but not `releng`. This should be fixed. 

Generally: 

* an **entire rewrite of the FAQ page** is planned :-)

----

Improvements to the main **Get FreeBSD** page (a.k.a. the huge yellow button at the home page) are ongoing. 

Whilst we're without a suitable FAQ page, two improvements last month ([1](https://github.com/freebsd/freebsd-doc/commit/5c8cc20333c4e87dd5d3a562ad87f5c18254d5fd?diff=split#diff-0b0b77802642b4009de6cc60f986b1f3d3dc56593e50ca3b1d41dae75da5402cR36-R38), [2](https://github.com/freebsd/freebsd-doc/commit/5c8cc20333c4e87dd5d3a562ad87f5c18254d5fd?diff=split#diff-0b0b77802642b4009de6cc60f986b1f3d3dc56593e50ca3b1d41dae75da5402cR144-R160)) may be of interest. Resulting texts: 

* <https://www.freebsd.org/where/#download>

> RELEASE versions of FreeBSD are the end result of _release engineering_.

* <https://www.freebsd.org/where/#helptest>

– includes an outline of the three branches. I could have added lowercase `main`, `stable` and `releng`, however (a) the page is already too long for my liking; and (b) a better place will be the rewritten FAQ.

Reply:
Thanks for the explanation @grahamperrin. Feel free to revisit the description to add better link(s) once they are available.